squad/frontend/project_settings.py

```diff
@@ -12,7 +12,7 @@
 class ProjectForm(forms.ModelForm):
     class Meta:
         model = Project
-        fields = ['name', 'is_public', 'is_archived', 'description']
+        fields = ['name', 'is_public', 'is_archived', 'description', 'enabled_plugins_list']
```

Comment:
could you also add timeouts?

Reply:
"notification timeout" and "wait for notification"? I also thought about "data rentention", what do you think?

Reply:
makes sense.

Reply:
added